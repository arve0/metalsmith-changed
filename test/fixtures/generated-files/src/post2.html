---
title: Second Post
date: 2015-10-02
---

This is the second post.