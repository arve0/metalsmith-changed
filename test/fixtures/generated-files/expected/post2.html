

This is the second post.